x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/schema_table.tsx

```diff
@@ -58,14 +58,15 @@ export const SchemaTable: React.FC = () => {
           <EuiTableRowCell align="right" />
         </EuiTableRow>
         {Object.entries(schema).map(([fieldName, fieldType]) => {
-          const isRecentlyAdded = unconfirmedFields.length && unconfirmedFields.includes(fieldName);
+          const isRecentlyAdded = unconfirmedFields.includes(fieldName);
+          const isMissingSubfields = fieldType === 'text' && incompleteFields.includes(fieldName);
```

Comment:
These changes require newly added API fields from enterprise search. How do we handle API changes and backwards compatibility? It is not a concern because we are bundling under a shared version?

Reply:
Correct, generally not a concern because we're a versioned product. It might be an issue for changes that are backwards incompatible, but extra fields rarely are.